tests/acceptance/features/webUIFiles/copy.feature

```diff
@@ -72,21 +72,24 @@ Feature: copy files and folders
     Given user "Brian" has been created with default attributes
     And user "Brian" has shared folder "simple-folder" with user "Alice" with "read" permissions
     And user "Alice" has logged in using the webUI
-    When the user tries to copy file "lorem.txt" into folder "simple-folder (2)" using the webUI
```

Comment:
I think we were supposed to keep this line as it is and make changes in the function that this step has been using.